src/App.jsx

```diff
@@ -66,7 +66,7 @@ function App() {
   }, [searchInput, loggedIn]);
 
   return (
-    <div>
+    <div className="App">
```

Comment:
I believe this is unnecessary since App is already added automatically with React. Sorry! Thank you for adding the overflow changes.

<img width="540" alt="image" src="https://github.com/awong-pdx/frontend-weather-dashboard/assets/74124296/7a12e4c5-8184-44e8-9e81-7aeb8b208498">

Reply:
Oh I didn't know that the App is automatically added! No problem, let me remove that!